docs/deploy-from-source-code.md

```diff
@@ -0,0 +1,94 @@
+# Deploy from source code
+
+## Platform from source code getting started
```

Comment:
Split to:
` ### Run from Visual Studio
and ` ### Run via dotnet
mind the different ports

Reply:
fixed